spec/views/users_index_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
```

Comment:
- [ ] Your tests are failing locally. Please ensure that all of the tests have passed successfully and request another review.

<img width="632" alt="image" src="https://user-images.githubusercontent.com/77190203/174135147-7bc065d7-7a55-411d-b59c-3543d9a58242.png">